next.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+  extends: ["./react.js"],
+  overrides: [
+    {
+      files: [
+        "**/__tests__/next/**/*.test.tsx",
+        "**/__tests__/pages/**/*.test.tsx",
+        "**/__tests__/pages/**/*.test.jsx",
+      ],
+      rules: {
+        // off because of testing library
+        "jest/valid-expect": "off",
+        "jest/expect-expect": "off",
+      },
+    },
+  ],
+  rules: {
+    "import/no-internal-modules": [
+      "error",
+      {
+        allow: [
+          "**/*.css",
+          "@testing-library/**/*",
+          "react-dom/*",
+          "**/configuration/**/*",
+          "**/configuration/*",
```

Comment:
these are app-specific and for example don't apply to most packages

Reply:
That's why they are in `nextjs` file.